be/test/plugin/plugin_test/source/test.zip.md5

```diff
@@ -1 +1,20 @@
+<!--
```

Comment:
Do not add license for this file

Reply:
Should I need to add documents to the exemption list?